desktop/main.js

```diff
@@ -20,7 +20,7 @@ app.on('ready', () => {
     resizable: true,
     frame: process.platform !== 'darwin',
     skipTaskbar: process.platform === 'darwin',
-    autoHideMenuBar: process.platform === 'darwin',
+    autoHideMenuBar: process.platform !== 'darwin',
```

Comment:
I also mended this, since this setting only applies to Windows & Linux, I suspect this was a typo (?)